lib/client/db/postgres/connstring.go

```diff
@@ -54,5 +54,12 @@ func GetConnString(c *profile.ConnectProfile, noTLS bool) string {
 		params = append(params,
 			fmt.Sprintf("sslmode=%v", SSLModeVerifyFull))
 	}
-	return fmt.Sprintf("%v?%v", connStr, strings.Join(params, "&"))
+	connStr = fmt.Sprintf("%v?%v", connStr, strings.Join(params, "&"))
+
+	// The printed connection string may get copy-pasted for execution. Add
+	// quotes to avoid "&" getting interpreted by terminals.
+	if printFormat {
+		connStr = fmt.Sprintf("%q", connStr)
```

Comment:
nit: `%q` gives us
```
%q	a double-quoted string safely escaped with Go syntax
```

If all we truly want to do is add quotes, then it may be best to use

```go
connStr = fmt.Sprintf(`"%s"`, connStr)
```

in order to avoid any unwanted Go syntax escapes. 

Additionally, I think 
```go
connStr = fmt.Sprintf(`'%s'`, connStr)
```

(single quoted) may be more appropriate for easy copy-pasting in the terminal, since that way the text is interpreted precisely as-is without the potential for mistaken [variable substitution](https://askubuntu.com/questions/20034/differences-between-doublequotes-singlequotes-and-backticks-on-comm)